coalib/parsing/CliParsing.py

```diff
@@ -65,6 +65,12 @@ def parse_cli(arg_list=None,
                                   origin,
                                   line_parser)
         else:
+            # if 'Bear' suffix is not already present then add it here
+            if arg_key == 'bears' and isinstance(arg_value, list):
+                arg_value = [
+                    Bear + 'Bear' if Bear and not Bear.endswith('Bear')
+                    else Bear for Bear in arg_value]
```

Comment:
What I'm just thinking, but I guess isn't really easy / possible, is to use our automatic settings parsing. We would introduce a new type `BearList`, that would automatically handle this. But like said, just an idea ;)

Reply:
where does this automatic settings parsing happen?